mkt/reviewers/templates/reviewers/includes/review_history.html

```diff
@@ -62,36 +62,33 @@
                   </td>
                 </tr>
               {% endif %}
-              {% if version.approvalnotes %}
```

Comment:
nice. i'm not sure if @eviljeff cares that these are at the top. In that case, you'd have to add some JS logic to render it here.